client/src/components/create-page/CreateProfile.js

```diff
@@ -0,0 +1,154 @@
+import { useState } from "react";
+import { Image } from "cloudinary-react";
+import "./CreateProfile.css";
+import Header from "../../components/Header";
+import { NavLink } from "react-router-dom";
+import { AiFillSave } from "react-icons/ai";
+import { GiDogHouse } from "react-icons/gi";
+import { addProfileToLocalStorage } from "../../utils/itemStorage";
+
+export default function CreateProfile() {
+  const [imageSelected, setImageSelected] = useState("");
+  const [imageType, setImageType] = useState("");
+  const [imagePublicId, setImagePublicId] = useState("");
+  const [imageId, setImageId] = useState("");
+  const [name, setName] = useState("");
+  const [bio, setBio] = useState("");
+  const [counter, setCounter] = useState(125);
+  const [clicked, setClicked] = useState(false);
+  const [info, setInfo] = useState({});
+
+  const uploadImage = (e) => {
+    e.preventDefault();
+    const formData = new FormData();
+    formData.append("file", imageSelected);
+    formData.append("upload_preset", "s2bkhsfz");
+
+    fetch("https://api.cloudinary.com/v1_1/dy1xpaosj/image/upload", {
+      method: "PUT",
+      body: formData,
+    })
+      .then((response) => response.json())
+      .then((result) => {
+        setImagePublicId(result.public_id);
+        setImageType(result.format);
+        setImageId(result.created_at);
+        setInfo(result);
+        console.log("Success:", result);
+        if (result.error.message === "Missing required parameter - file") {
+          alert("Please select a picture to upload.");
+        }
+      })
+      .catch((error) => {
+        console.error("Error:", error);
```

Comment:
does this just log the error in the console? Maybe you could do an alert with an error message or direct to an error page